index.html

```diff
@@ -205,7 +205,7 @@ <h3 class="cardHeader aboutmeHeader">Skills</h3>
                 <li>Terminal</li>
             </ul>
         </div>
-
+    </div>
```

Comment:
- [ ] Add the min-width and max-width CSS property for your cards to avoid this issue (as you can see in the red color in the image below) in your front-end and also when someone is changing the window size.
- [ ] Add more margin/padding between the cards section and the content me section (as you can see in the yellow color in the image below).
- [ ] [OPTIONAL] For the skills card you can try to use the right image provided by the Figma design 😉 

Figma design | Yours

![image](https://user-images.githubusercontent.com/60164239/130099240-cb5ce101-9a86-476f-8a1b-9148a5a78086.png)